app/uk/gov/hmrc/economiccrimelevyregistration/controllers/NotableErrorController.scala

```diff
@@ -75,4 +76,10 @@ class NotableErrorController @Inject() (
   def assistantCannotRegister: Action[AnyContent] = authoriseAssistantsAllowed { implicit request =>
     Ok(assistantCannotRegisterView())
   }
+
+  def organisationAlreadyRegistered(eclRegistrationReference: String): Action[AnyContent] =
```

Comment:
just wondering if for consistency we should name the query param here: `eclReferenceNumber` instead, since that's what we describe it as to users everywhere else and the user will be able to see this in the URL.

or perhaps make it a URI parameter instead, that way it doesn't have to be named